lib/district_repository.rb

```diff
@@ -14,47 +12,59 @@ def find_by_name(name)
     pair = districts.detect do |_key, value|
       name.strip.upcase == value.name.upcase
     end
+
     pair && pair[1]
   end
 
-  def find_all_matching(name_fragment)
+  def find_all_districts_matching_name_fragment(name_fragment)
     matching = districts.select do |_key, value|
       value.name.upcase.include?(name_fragment.upcase)
     end
+
     matching.values
   end
 
-  def parser(contents)
+  def kindergarten_participation_access_hash
```

Comment:
Throughout the app there were mixed hash assignments to get data from the CSV store and push it to the District or Enrollment classes. Having a consistent pattern throughout the application that is abstractable (in order to obtain the key and values necessary to parse the data) for the sake of DRY when we parse more than just these two files.